docs/Maintainer-Guidelines.md

```diff
@@ -37,6 +37,10 @@ forever. Nobody really checks if they are necessary or not. Use the
 Depend on as little stuff as possible. Disable X11 functionality if possible.
 For example, we build Wireshark, but not the heavy GUI.
 
+For [some formulae](https://github.com/Homebrew/homebrew-core/search?q=%22homebrew%2Fmirror%22&unscoped_q=%22homebrew%2Fmirror%22),
+we mirror the tarballs to our own BinTray. To do this, run
+`brew mirror <formula>` after a version bump PR is merged.
```

Comment:
I'd say `with a local checkout of a PR before it is merged`.

If you fancy the challenge (or even just writing up an issue!) this would be a great thing to do automatically at bottle publish time.

Reply:
Luckily `brew` is clever enough to [handle the "already merged" case](https://gist.github.com/issyl0/751f30168679f2e275a6bb80e3f41a49), but if anyone tries to upgrade in the few seconds before that happens that may well be suboptimal user experience, yeah. I'll take a look!

Reply:
I don't have enough brain for that challenge today (maybe later in the week 😄), but I [wrote up an issue](https://github.com/Homebrew/brew/issues/7708).

Reply:
Works for me!



> ries to upgrade in the few seconds before that happens that may well be suboptimal

Yeh, that was my thinking 👍

Reply:
I applied your suggested change.